bananas_server/application/bananas_server.py

```diff
@@ -41,6 +41,34 @@ def _send_content_entry(self, source, content_entry):
             tags=content_entry.tags,
         )
 
+    def _safe_name(self, name):
+        new_name = ""
+
+        for letter in name:
+            if (
+                (letter >= "a" and letter <= "z")
+                or (letter >= "A" and letter <= "Z")
+                or (letter >= "0" and letter <= "9")
+                or letter == "."
```

Comment:
Do not allow "." in first position?

Reply:
Not disagreeing, but there are more things I would like to fix. For now I did what musa did:
https://github.com/OpenTTD/musa/blob/68bf7d1f1983f028ec2b2a775fe421618d6a4c0c/misc.py#L91
(only written in better Python). I guess we can clean up some more .. like no starting with _ or ending with it :D